client/header/activity-panel/panels/inbox/index.js

```diff
@@ -178,11 +190,13 @@ export default compose(
 		const notes = getNotes( inboxQuery );
 		const isError = Boolean( getNotesError( inboxQuery ) );
 		const isRequesting = isGetNotesRequesting( inboxQuery );
+		const isUndoRequesting = isUndoDismissRequesting();
```

Comment:
Based on the ways this get compared to strings in the component, does it make sense to split this up into 2 or 3 variables like: `isUndoRequesting`, `isDismissUndoRequesting`, `isDismissAllUndoRequesting`?

Reply:
You are completely right, I like the idea. It was done in the commit [8b64bb9](https://github.com/woocommerce/woocommerce-admin/pull/4379/commits/8b64bb904ab47025dc75cda8040e01eb8c21156d)